homeassistant/components/recorder/db_schema.py

```diff
@@ -740,6 +741,7 @@ def from_stats(
             start=None,
             start_ts=stats["start"].timestamp(),
             mean=stats.get("mean"),
+            circular_mean=stats.get("circular_mean"),
```

Comment:
Why would we introduce a new field for this?

Reply:
@emontnemery suggested adding a new field for it

Reply:
I much prefer adding a new column so it's never ambiguous what kind of data is in the column. If we reuse the existing column, we can't really guarantee it doesn't have other data than circular mean, unless we destroy all past data when entities change, entity id is reused etc.

Reply:
Yeah ok... 👍